src/Sparrow/Binary/Numbers.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Sparrow.Binary;
+
+public static class Numbers
```

Comment:
Why not use?

https://docs.microsoft.com/uk-ua/dotnet/api/system.buffers.text.base64.encodetoutf8?view=net-6.0#system-buffers-text-base64-encodetoutf8(system-readonlyspan((system-byte))-system-span((system-byte))-system-int32@-system-int32@-system-boolean)

Reply:
Base64 is a much more involved format. That would require a method call, in here this just gets inlined as individual instructions. This is just the number in hexadecimal format, so I can control the size it will have.